### PR TITLE
Kotlin 1.4.0-rc

### PR DESCRIPTION
**Pull request by mpetuska**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -19,9 +19,11 @@ android.useAndroidX=true
 android.enableJetifier=true
 # Kotlin code style for this project: "official" or "obsolete":
 kotlin.code.style=official
+kotlin.native.ignoreDisabledTargets=true
+kotlin.mpp.enableGranularSourceSetsMetadata=true
 
 GROUP=org.reduxkotlin
-VERSION_NAME=0.5.3
+VERSION_NAME=0.5.4-SNAPSHOT
```
Comment (patjackson52) [inline gradle.properties:26]: Adding SNAPSHOT is not needed.  There is a env var that is set to make snapshot builds.  CI will take care of adding the "-SNAPSHOT" appendage.  Releases are published only on tags.

File: redux-kotlin-threadsafe/build.gradle [hunk 2]
```diff
@@ -72,7 +70,6 @@ kotlin {
         }
         jsMain {
             dependencies {
```
Comment (patjackson52) [inline redux-kotlin-threadsafe/build.gradle:72]: Nice clean up, thanks!  Lets remove the empty dependencies {}

File: gradle.properties (hunk 1 above)
Comment (mpetuska) [reply gradle.properties]: Reverted

File: redux-kotlin-threadsafe/build.gradle (hunk 2 above)
Comment (mpetuska) [reply redux-kotlin-threadsafe/build.gradle]: Cleaned up

**Comment by mpetuska** [comment]
@patjackson52 in regards to your latest comment looks like we'll have to live with those warnings as when I specify js flavour, gradle breaks with `Caused by: org.gradle.api.internal.tasks.DefaultTaskContainer$DuplicateTaskException: Cannot add task 'clean' as a task with that name already exists.` error

**Pull request merged**